Bigtable/src/Connection/Grpc.php

```diff
@@ -0,0 +1,544 @@
+        $appProfileId = $this->pluck('appProfileId', $args);
+        $appProfile = $this->pluck('appProfile', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'createAppProfile'], [
+            $parent,
+            $appProfileId,
+            $appProfile,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function getAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'getAppProfile'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function listAppProfiles(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'listAppProfiles'], [
+            $parent,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function updateAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        $appProfile = $this->pluck('appProfile', $args);
+        $updateMask = $this->pluck('updateMask', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'updateAppProfile'], [
+            $appProfile,
+            $updateMask,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function deleteAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        $ignoreWarnings = $this->pluck('ignoreWarnings', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'deleteAppProfile'], [
+            $name,
+            $ignoreWarnings,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function getIamPolicy(array $args)
+    {
+        $resource = $this->pluck('resource', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'getIamPolicy'], [
+            $resource,
+            $this->addResourcePrefixHeader($args, $resource)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function setIamPolicy(array $args)
+    {
+        $resource = $this->pluck('resource', $args);
+        $policy = $this->pluck('policy', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'setIamPolicy'], [
+            $resource,
+            $policy,
+            $this->addResourcePrefixHeader($args, $resource)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function createTable(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        $tableId = $this->pluck('tableId', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'createTable'], [
```

Comment:
This method should allow users to add column families.  I really would prefer if we implemented this later.